docs/content/build.md

```diff
@@ -0,0 +1,38 @@
+**Warning:** The [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+
+-----
+## Building your workers
+
+Before running a deployment of your game, you need to configure your [workers](https://docs.improbable.io/reference/latest/shared/glossary#worker), configure the build of your workers and also build the workers. This applies to both server-workers and client-workers.
```

Comment:
Needs a link to the deploy page (the currently non-existent glossary would be the best option, though).

Would the worker link be better linking to the new worker intro page in this repo? (I think it would.) We can link to the current worker.md as a temporary measure. And then put the new worker generic info into the worker.md file?)

Reply:
"..and also build your workers."  <-- we use "you" throughout and should stick with.